### ISSUE TITLE
Create One Touch Accounts in AUA

### ISSUE THREAD

**Issue by ghost** [nwfsc-fram/boatnet#736]
Need to figure out how to have a single account for a given captain to support One Touch

Placeholder ticket for resolving the issue of multiple captain accounts in AUA.

FYI:  @wsmith-nwfsc , @sethgerou-noaa

**Comment by SaOgaz-NOAA** [comment]
@sethgerou-noaa, is this basically the thing about captiains having separate apexuseradmin accounts for the permits system and for boatnet?

**Comment by sethgerou-noaa** [comment]
@SaOgaz-NOAA Correct. Though as I understand it, we're able to grant an existing AUA user access to a new app and roles, so the task is to come up with a streamlined means of checking for an existing account, adding access to 'BOATNET_OBSERVER' app, and granting some roles.
